lib/server.js

```diff
@@ -285,9 +285,8 @@ export default class Server {
     }
 
     if (this.isTest()) {
-      if (config.testConfig) {
```

Comment:
All this code did was conditionally load the config. How does this effect timing?

Reply:
I was confused by this too, but `loadConfig` is where the timing gets set, for some reason.

Reply:
I wonder if https://github.com/miragejs/miragejs/blob/d26b777ee9bcd6ba6f4e89c9df6c509e54476c2b/lib/server.js#L345 should be moved outside of `loadConfig` and into this `if (this.isTest())` conditional.

Reply:
Ahh, I believe this line should be 
```
this.timing = this.isTest() ? undefined : this.timing; 
```

Reply:
It's easy to get turned around in here.  😅